externalize_tests_tmp.go

```
@@ -0,0 +1,115 @@
+			cursor.Replace(&ast.SelectorExpr{X: ast.NewIdent(*alias), Sel: ast.NewIdent(identifier.Name)})
+			return false
+		}, nil)
+
+		file.Name.Name = pkg.Name + "_test"
+		astutil.AddNamedImport(pkg.Fset, file, *alias, pkg.PkgPath)
+		output, err := os.Create(pkg.CompiledGoFiles[index])
+		if err != nil {
+			panic(err)
+		}
+		if err := format.Node(output, pkg.Fset, file); err != nil {
+			output.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `output.Close` is not checked

(errcheck)